Tests/Fixtures/SecurityTestController.php

```diff
@@ -28,11 +28,9 @@ public function scopeAction(): Response
 
     public function rolesAction(): Response
     {
-        $roles = $this->get('security.token_storage')->getToken()->getRoles();
-
-        $roles = array_map(function (Role $role): string {
-            return $role->getRole();
-        }, $roles);
+        /** @var TokenStorage $tokenStorage */
+        $tokenStorage = $this->get('security.token_storage');
```

Comment:
This change seems unrelated to SF 5.

Reply:
so this change is needed because TokenStorage changed in SF5, where you no-longer call role->getRole. reverting this change will actually cause an Error in the unit tests. So I do think  we need to keep this one around.